app/packs/src/apps/mydb/elements/list/ElementsTableSampleEntries.js

```diff
@@ -107,6 +107,20 @@ const showDecoupledIcon = (sample) => (sample.decoupled ? (
   </OverlayTrigger>
 ) : null);
 
+const showInventoryLabelIcon = (sample) => (sample.inventory_sample && sample.xref.inventory_label ? (
```

Comment:
then the condition could be simply `sample.inventory_label` if defined in the model

Reply:
done